### PR TITLE
winch: Aarch64 sub and mul

### PR DESCRIPTION
**Pull request by jimmyhmiller**
I just filled in some `todo!` for sub and mul on aarch64. I copied the file tests from x86. It should be fairly straight forward, but of course there is a chance I missed something obvious.

@saulecabrera You mentioned fuzz testing being your main approach now. I wasn't sure how to do that, but would happily go run some if you can point me in the right direction.

### CODE REVIEW

File: winch/filetests/filetests/aarch64/i64_sub/unsigned_with_zero.wat [hunk 1]
```diff
@@ -0,0 +1,22 @@
+;;! target = "aarch64"
+
+(module
+    (func (result i64)
+        (i64.const 1)
+     	(i64.const 0)
+    	(i64.add)
```
[the remark this answers is not in the document]
Comment (saulecabrera) [reply winch/filetests/filetests/aarch64/i64_sub/unsigned_with_zero.wat:7]: Thanks for pointing this out -- it should definitely be `sub`.

**Comment by saulecabrera** [comment]
> You mentioned fuzz testing being your main approach now. I wasn't sure how to do that, but would happily go run some if you can point me in the right direction.

Yeah, unfortunately you won't be able to get local fuzzing working for  `aarch64` working since it will at least require support for `call`, which we don't have yet. It shouldn't be to complex to get going though with all the recent work that we've done for supporting function calls and trampolines.

**Review by saulecabrera — APPROVED** [review]
Looks good to me; I left a comment regarding one file test that includes x64 output, but the specified architecture is aarch64.

Would you also be willing to fix the x64/aarch64 subtraction test that you pointed out here https://github.com/bytecodealliance/wasmtime/pull/6468/files#r1207028404? 

I think that after those fixes, we can land this change.

**Comment by jimmyhmiller** [comment]
@saulecabrera Fixed. Sorry for the late responses on these. My notifications were still going to my old work email. Fixed now.

**Comment by saulecabrera** [comment]
Thanks for the fixes! There's a failure in the updated `sub` tests for `x64`, because the output doesn't match (as expected), you should be able to fix that by running `WINCH_TEST_BLESS=1 cargo run -- test` inside the `winch` directory.

**Comment by jimmyhmiller** [comment]
Yeah, sorry my bad. Totally thought I ran that a second time. Fixed now.

**Pull request merged**